recipes/r-fedmatch/meta.yaml

```
@@ -0,0 +1,106 @@
+{% set version = '2.0.6' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-fedmatch
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/fedmatch_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/fedmatch/fedmatch_{{ version }}.tar.gz
+  sha256: e2e143c3cdda215920f76abf1e1528e2ec000bbe3bb52a4d51559989c3909419
+
+build:
+  merge_build_host: True  # [win]
+  missing_dso_whitelist:
+    - '*/R.dll'  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ stdlib("c") }}
```

A little adjustment for win:

```suggestion
    - {{ stdlib("c") }}        # [not win]
    - {{ stdlib("m2w64_c") }}  # [win]
```